### PR TITLE
feat: add methods to add and delete a data integration

### PR DESCRIPTION
**Pull request by romainrousseaux**

### CODE REVIEW

File: src/kili/use_cases/cloud_storage/__init__.py [hunk 1]
```diff
@@ -303,3 +306,60 @@ def _validate_data_differences(
...
+        aws_role_external_id=aws_role_external_id,
+        azure_connection_url=azure_connection_url,
+        azure_is_using_service_credentials=azure_is_using_service_credentials,
+        azure_sas_token=azure_sas_token,
+        azure_tenant_id=azure_tenant_id,
+        gcp_bucket_name=gcp_bucket_name,
+        include_root_files=include_root_files,
+        internal_processing_authorized=internal_processing_authorized,
+        name=name,
+        organization_id=OrganizationId(""),
```
Comment (PierreLeveau) [inline src/kili/use_cases/cloud_storage/__init__.py:349]: I guess it is ignored backend side but kept required in the GraphQL schema?

**Review by PierreLeveau — APPROVED** [review]
nice to have : a e2e test to make sure we are compatible with the GraphQL endpoint.

**Comment by PierreLeveau** [comment]
you did not add a method in `kili/presentation/client/cloud_storage.py`, so this won't be part of the SDK module (maybe you forgot to push a file?).

File: src/kili/use_cases/cloud_storage/__init__.py (hunk 1 above)
Comment (romainrousseaux) [reply src/kili/use_cases/cloud_storage/__init__.py:349]: indeed

**Review by PierreLeveau — APPROVED** [review]

**Comment by Jonas1312** [comment]
https://github.com/kili-technology/kili-python-sdk/actions/runs/7460662342

**Review by Jonas1312 — APPROVED** [review]

**Review by Jonas1312 — APPROVED** [review]
merge once e2e tests pass :)

**Pull request merged**